### DISCUSSION TITLE
[@types/aws-lambda] - cognito handler - Missing types for Custom SMS sender Lambda trigger

### DISCUSSION THREAD

**Discussion by jvliwanag** [DefinitelyTyped/DefinitelyTyped#58559] [Issues with a @types package]
Types are missing for Custom SMS sender as documented in: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-email-sender.html .

[a comment by an automated account is not shown]

**Reply by darbio** [reply]
Feel free to add them in and submit a PR.
